subprojects/docs/src/docs/userguide/upgrading_version_5.adoc

```diff
@@ -52,6 +52,13 @@ Creating `SignOperation` instances directly is now deprecated. Instead, the meth
 
 This will become an error in Gradle 6.0.
 
+==== Declaring an incremental task without outputs
+
+Declaring an <<custom_tasks.adoc#incremental_tasks,incremental task>> without declaring outputs is now deprecated.
+Declare file outputs or use `outputs.upToDateWhen { true }` instead.
```

Comment:
I think here we should also say `upToDateWhen()`.

Reply:
Done.